lib/giraph/schema.rb

```diff
@@ -0,0 +1,46 @@
+module Giraph
+  # GraphQL::Schema wrapper allowing decleration of
+  # resolver objects per op type (query or mutation)
+  class Schema < GraphQL::Schema
```

Comment:
The risk you take when you subclass code that you don't control is that something in the implementation of the base class will change and your specialization of it will somehow cause it to break in the future. I agree that in this case, there's little to no risk of that, since it's not overriding any existing functionality, it's just doing some additional work before delegating to the super class. In my opinion, that's another argument in favor of using delegation here, since subclassing is completely unnecessary.